yt_dlp/extractor/youtube.py

```diff
@@ -4480,14 +4480,15 @@ def process_language(container, base_url, lang_code, sub_name, query):
                             if mobj:
                                 info[mobj.group('type') + '_count'] = str_to_int(mobj.group('count'))
                                 break
-            sbr_tooltip = try_get(
-                vpir, lambda x: x['sentimentBar']['sentimentBarRenderer']['tooltip'])
-            if sbr_tooltip:
-                like_count, dislike_count = sbr_tooltip.split(' / ')
-                info.update({
-                    'like_count': str_to_int(like_count),
-                    'dislike_count': str_to_int(dislike_count),
-                })
+
+            match = re.search(
+                r'\b(\d+)\b',
+                vpir['videoActions']['menuRenderer']['topLevelButtons'][0]
+                ['segmentedLikeDislikeButtonViewModel']['likeButtonViewModel']
+                ['likeButtonViewModel']['toggleButtonViewModel']['toggleButtonViewModel']
+                ['defaultButtonViewModel']['buttonViewModel']['accessibilityText'])
+            info['like_count'] = str_to_int(match.group(1))
```

Comment:
extracting metadata such as `like_count` should be completely non-fatal, so we want to use `traverse_obj` and `parse_count`
```suggestion
            info['like_count'] = traverse_obj(vpir, (
                'videoActions', 'menuRenderer', 'topLevelButtons', ...,
                'segmentedLikeDislikeButtonViewModel', 'likeButtonViewModel', 'likeButtonViewModel',
                'toggleButtonViewModel', 'toggleButtonViewModel', 'defaultButtonViewModel',
                'buttonViewModel', 'accessibilityText', {parse_count}), get_all=False)
```